packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -112,13 +112,11 @@ class _CodeViewState extends State<CodeView> with AutoDisposeMixin {
 
   @override
   void dispose() {
-    super.dispose();
-
     gutterController.dispose();
     textController.dispose();
-
     widget.controller.scriptLocation
         .removeListener(_handleScriptLocationChanged);
+    super.dispose();
```

Comment:
@pq is there a lint we should be opting into to catch calling super first instead of last?

Reply:
No.  We had one (now deprecated) for enforcing calling super last in constructor initializers (https://dart-lang.github.io/linter/lints/super_goes_last.html).  Rolling one for dispose calls is an interesting idea.  Especially if we can generalize somehow.